src/delay.js

```diff
@@ -53,10 +55,7 @@ define(function (require) {
    *  </code></div>
    */
   p5.Delay = function() {
-    this.ac = p5sound.audiocontext;
-
-    this.input = this.ac.createGain();
-    this.output = this.ac.createGain();
+  	p5.Effect.call(this);
```

Comment:
`p5.Effect` works because of the order of app.js, but it's safer not to depend on that and to instead require dependencies explicitly. Try to use `Effect`, since that's what we required on line 6. Same goes for `Filter`, if you feel like cleaning up the lines that refer to `p5.Filter` (if you don't get to it in this PR, I will be happy to)